master/paths.html

```diff
@@ -340,33 +340,13 @@ <h3 id="PathDataMovetoCommands">The <strong>"moveto"</strong> commands</h3>
 <h3 id="PathDataClosePathCommand">The <strong>"closepath"</strong> command</h3>
 
 <p>The "closepath" (<strong>Z</strong> or <strong>z</strong>)
-  must end the current subpath by connecting it back to its <a>initial point</a>
-  in either of two ways:
+  ends the current subpath by connecting it back to its <a>initial point</a>.
+  An automatic
+  straight line is drawn from the current point to the <a>initial point</a>
+  of the current subpath. This <a>path segment</a> may be of zero
+  length.
```

Comment:
The original text also said what the next initial point is when "closepath" is not followed by a moveto.

Reply:
Also, IMO it is important to keep the original sentence that a lineto to the initial point is not the same as specifying a closepath.

Maybe we should copy those sentences over: https://www.w3.org/TR/SVG11/paths.html#PathDataClosePathCommand

Reply:
The sentences have now been copied.